x-pack/plugins/secops/public/components/autocomplete_field/index.tsx

```diff
@@ -132,6 +132,7 @@ export class AutocompleteField extends React.Component<
         break;
       case 'Escape':
         evt.preventDefault();
+        evt.stopPropagation();
```

Comment:
Awesome! Thanks for this fix! :) Nice to not have the timeline close on you when dismissing the suggestions.